bundles/org.openhab.core.voice/src/test/java/org/openhab/core/voice/internal/cache/AudioStreamCacheWrapperTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.voice.internal.cache;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.openhab.core.audio.AudioStream;
+import org.openhab.core.voice.TTSException;
+
+/**
+ * Test the wrapper stream in the cache system
+ *
+ * @author Gwendal Roulleau - Initial contribution
+ */
+@NonNullByDefault
+public class AudioStreamCacheWrapperTest {
+
+    /**
+     * Test the read() method
+     *
+     * @throws IOException
+     */
+    @Test
+    public void cacheWrapperStreamTest() throws IOException {
+        TTSResult mockedTTSResult = Mockito.mock(TTSResult.class);
+        AudioStreamSupplier mockedAudioStreamSupplier = Mockito.mock(AudioStreamSupplier.class);
```

Comment:
You can make these mocks members of the class:

```
private @Mock @NonNullByDefault({}) TTSResult ttsResultMock;
```

and they get mocked automatically if you annotate the class with

```
@ExtendWith(MockitoExtensions.class)
```

Reply:
Done, thanks